Calibration/EcalCalibAlgos/interface/EcalPhiSymRecHit.h

```diff
@@ -0,0 +1,64 @@
+#ifndef ECAL_PHISYM_RECHIT_H
+#define ECAL_PHISYM_RECHIT_H
+
+/** \class EcalPhiSymRecHit
+ * 
+ * Dataformat dedicated to Phi Symmetry ecal calibration
+ * 
+ * Note: SumEt array ordering:
+ *       0         - central value
+ *       1<->N/2   - misCalib<1
+ *       N/2+1<->N - misCalib>1
+ *
+ * Original Author: Simone Pigazzini (2022)
+ */
+
+#include <vector>
+#include <cassert>
+
+#include "DataFormats/DetId/interface/DetId.h"
+#include "DataFormats/EcalDetId/interface/EBDetId.h"
+#include "DataFormats/EcalDetId/interface/EEDetId.h"
+#include "DataFormats/EcalDetId/interface/EcalSubdetector.h"
+
+class EcalPhiSymRecHit {
+public:
+  //---ctors---
+  EcalPhiSymRecHit();
+  EcalPhiSymRecHit(uint32_t id, unsigned int nMisCalibV, unsigned int status = 0);
+  EcalPhiSymRecHit(uint32_t id, std::vector<float>& etValues, unsigned int status = 0);
+
+  //---dtor---
+  ~EcalPhiSymRecHit();
+
+  //---getters---
+  inline uint32_t GetRawId() const { return id_; };
+  inline unsigned int GetStatusCode() const { return chStatus_; };
+  inline uint32_t GetNhits() const { return nHits_; };
+  inline unsigned int GetNSumEt() const { return etSum_.size(); };
+  inline float GetSumEt(int i = 0) const { return etSum_[i]; };
+  inline float GetSumEt2() const { return et2Sum_; };
+  inline float GetLCSum() const { return lcSum_; };
+  inline float GetLC2Sum() const { return lc2Sum_; };
```

Comment:
Ideally (by [CMSSW naming rules](https://cms-sw.github.io/cms_coding_rules.html#2--naming-rules-1)) for getter methods, using the value name is preferred, e.g. `rawId()`, `statusCode()`...
Not critical, but since it's a new class maybe it's better already start with the correct naming.